### ISSUE TITLE
Add Related Items (different mold types) to sidebar

### ISSUE THREAD

**Issue by BrickSouth** [rgriebl/brickstore#67]
I tried attaching a screenshot, but it won't process the file for some reason...anyway see the link below for an example. There is a section of some catalog entries called "Related Items" that deals mainly with other parts that are of a similar mold type. Having this section on the sidebar in BrickStore would be helpful for those of us who sell used and differentiate between mold types.
 
https://www.bricklink.com/v2/catalog/catalogitem.page?P=3004#T=P

**Comment by ZZJHONS** [comment]
It would be helpful but I don't know if BrickStore could access the relationship information.

https://www.bricklink.com/catalogRel.asp

**Comment by paramecie** [comment]
I'm not 100% sure, but I would say:

* cannot get this info through the APIs

* could only get this with downloading Catalog page and parsing it

In all cases, that means that somewhere a BS server should get the infos permanently to make a database that BS applications would download.

But even with 1000 downloads per day (o be safe), this would need 63 days (2 months) to browse the whole catalog - with 95% of the downloads not meaning anything (few parts have related items).

There could be another system where BS applications would upload infos downloaded to a BS server, and it would build aggregated datas sent back to all apps, but it's even more complicated.

In short - and IMO - forget it, sorry.

Robert may have a different idea, though.
And you still have the Catalog info you can right click on...

**Comment by BrickSouth** [comment]
No problem if not feasible, just a thought!

**Comment by paramecie** [comment]
... and no problem as suggesting!

It's just many BrickLink infos are only available to BrickLink users (for obvious reasons).

Circumventing this is complicated, may be long to implement, fragile, and still as being forbidden by BrickLink, they can then unauthorize IP, signature, or anything related to BS, including forum posts... which nobody wish ;-)
